src/biogeochem/CNFireLi2014Mod.F90

```diff
@@ -42,6 +43,7 @@ module CNFireLi2014Mod
   use PatchType                          , only : patch                
   use CNFireMethodMod                    , only : cnfire_method_type
   use CNFireBaseMod                      , only : cnfire_base_type, cnfire_const
+!  use CNVegMatrixType              , only : cnveg_cnmatrix_type
```

Comment:
Please remove this commented out line. Also, please remove unnecessary commented out lines in other places for your added codes.